docs/background/roadmap.rst

```diff
@@ -232,9 +232,6 @@ features.
 Platforms
 ---------
 
-Toga currently has good support for Cocoa on OS X, GTK+, and iOS.
-Proof-of-concept support exists for Windows Winforms. Support for a more
-modern Windows API would be desirable.
-
-In the mobile space, it would be great if Toga supported Android, Windows
-Phone, or any other phone platform.
+Toga currently has good support for Cocoa on macOS, GTK+, Winforms on Windows,
```

Comment:
While we're on this, we might as well remove the "+" from GTK, which was [officially renamed in 2019](https://mail.gnome.org/archives/gtk-devel-list/2019-February/msg00000.html). It also appears in several other places in the docs.

Also, for consistency with the other items in the list, it should say "GTK on Linux". I know GTK is available for Windows and Mac as well, but it certainly wouldn't be accurate to say we have "good support" for that.